### ISSUE TITLE
Python version requirements

### ISSUE THREAD

**Issue by skoslowski** [larose/utt#74]
I am (happily) using utt 1.22 with Python 3.7. It mostly works perfectly, however `utt report --week` fails with an
```
AttributeError: type object 'datetime.date' has no attribute 'fromisocalendar'
```
This function was added in Python 3.8.

Not sure if this is a mistake or if Python 3.8 is intended to be the minimum required version, in which case may I suggest to add a `python_requires` setting to `setup()` and update `Pipfile`

Thanks for putting this package together!

**Comment by larose** [comment]
Thanks for the bug report @skoslowski. That's indeed a mistake, the intent is to support Python 3.7. I'll check how to backport that feature to Python 3.7 and make a new release. I should be able to do this in the next few weeks.

**Comment by ukrutt** [comment]
Sorry for the delay.

I believe I'm the one who originally added this feature.  It relies on some functions in `fromisocalendar`, which was added in 3.8.  I tried to do it without `fromisocalendar`, but that would have necessitated a lot of machinery -- and at the time I did not see the point of reinventing the wheel.  

Thus, the `--week` switch, for now, is only available if you run Python 3.8.

**Comment by larose** [comment]
This has been fixed in 1.23!

**Comment by ukrutt** [comment]
Thanks @larose !
